testData/annotation.semantic/ImplementExternInterface.hx

```diff
@@ -0,0 +1,7 @@
+class Test1 implements ExternInterfaceTest {
+  public function fooExtern(b:Int):Int return 2;
+}
+
+extern interface ExternInterfaceTest {
+  function fooExtern(b:Int):Int;
+}
```

Comment:
I don't know much about annotation tests, but what it does, performs some check?